helpers.go

```diff
@@ -297,3 +297,20 @@ func updatePatchCmd(zypperCmd string, ctx *cli.Context) {
 		exitWithCode(1)
 	}
 }
+
+// joinAsArray joins the given array of commands so it's compatible to what is
+// expected from a dockerfile syntax.
+func joinAsArray(cmds []string, emptyArray bool) string {
+	if emptyArray && len(cmds) == 0 {
+		return ""
+	}
+
+	str := "["
+	for i, v := range cmds {
+		str += "\"" + v + "\""
+		if i < len(cmds)-1 {
+			str += ", "
+		}
+	}
+	return str + "]"
```

Comment:
I will leave it as it is, since I feel that it's more clear what's going on with a traditional for loop :)